protocol/v2/ssv/validator/committee.go

```diff
@@ -324,6 +324,7 @@ func (c *Committee) unsafePruneExpiredRunners(logger *zap.Logger, currentSlot ph
 
 	for slot := range c.Runners {
 		if slot <= minValidSlot {
+			logger = logger.With(fields.DutyID(fields.FormatCommitteeDutyID(c.Operator.Committee, c.BeaconNetwork.EstimatedEpochAtSlot(slot), slot)))
```

Comment:
did you check this on stage? wouldn't this add 2 duty_id fields since it already has dutyID field?

Reply:
checked with e2e

Reply:
and can you confirm my question?

Reply:
Did this by passing down a new logger